### ISSUE TITLE
failed! SQLSTATE[42S02]: Base table or view not found: 1146 Table 'modx_system_settings' doesn't exist (SQL: select * from `modx_system_settings`)

### ISSUE THREAD

**Issue by q2apro** [evolution-cms/evolution#2111]
When I try to install EvoCMS 3.1.10 and click from the last page (/install/index.php?action=summary) on "Next":

![image](https://user-images.githubusercontent.com/6764548/201678702-04abb90c-3118-4a8f-ac3f-8abf262ae40e.png)

I get this error message:

> failed! SQLSTATE[42S02]: Base table or view not found: 1146 Table 'modx_system_settings' doesn't exist (SQL: select * from `modx_system_settings`)

How to fix this and make the installer work? (There are not tables in the DB created)

**Comment by fourroses666** [comment]
You might consider to update to 3.1.12 CE which is the latest version. This CE version is going to be updated in the future while I doubt it will be for this repositery.
https://github.com/evocms-community/installer

**Comment by Dmi3yy** [comment]
> You might consider to update to 3.1.12 CE which is the latest version. This CE version is going to be updated in the future while I doubt it will be for this repositery.
> https://github.com/evocms-community/installer

Correct name for this fork: Russian Edition

**Comment by bossloper** [comment]
Yes, my moral compass has reached breaking point. This is all so sad.

**Comment by Dreamer0x01** [comment]
> > You might consider to update to 3.1.12 CE which is the latest version. This CE version is going to be updated in the future while I doubt it will be for this repositery.
> > https://github.com/evocms-community/installer
> 
> Correct name for this fork: Russian Edition

No, you're wrong. The correct name of this fork is **C**ommunity **E**dition.

The community is represented by participants from several countries and is absolutely not politicized.

**Comment by Dmi3yy** [comment]
> > > You might consider to update to 3.1.12 CE which is the latest version. This CE version is going to be updated in the future while I doubt it will be for this repositery.
> > > https://github.com/evocms-community/installer
> > 
> > 
> > Correct name for this fork: Russian Edition
> 
> No, you're wrong. The correct name of this fork is **C**ommunity **E**dition.
> 
> The community is represented by participants from several countries and is absolutely not politicized.

i would agree if they didn't steal the repository was created by me: 

https://github.com/extras-evolution

thank god they didn't manage to steal the main repository. But they try.

**Comment by Dmi3yy** [comment]
So it's not a question of politics, but of a normal attitude.  So I'm not ready to do a common thing with those who steal even what is free.  And it's not about politics anymore.

**Comment by Dmi3yy** [comment]
In general, I can only wish you success.  And on a personal level, I limited myself to the fact that I don’t want to have any business, including in open source, with those with whom my life values ​​​​do not match.

**Comment by Dmi3yy** [comment]
I also note that calling the war politics is not entirely correct.  Politics don't kill people.

**Comment by Dreamer0x01** [comment]
It's just a community of developers who have nothing to do with war or politics, but are just engaged in website development. And they continue to support and complement a good and convenient cms as much as they can.

**Comment by Dmi3yy** [comment]
Read what collective responsibility is and that even a simple developer who is in an aggressor country, willing or not, supports his country by paying taxes. Not to mention those who went to fight in another country without even understanding what they are fighting for. 

And yes, it's not politicians who went to war, but ordinary people. 

In general, I have heard your position and I see no point in continuing this dialogue.  

I think you already know everything I wrote above and I have not said anything new. 

Translated with www.DeepL.com/Translator (free version)

**Comment by Dreamer0x01** [comment]
We did not assume and did not sign this responsibility for the actions of other people. Like many other developers. This is someone's convenient delusion, picked up by politicians, which allows you to shift the blame of politicians to innocent people.

**Comment by Dmi3yy** [comment]
In any normal country it is the people who decide what politicians do, and if a politician is not to their liking, then another one comes to replace him. 

And the reason can be very trivial, as happened in England, the prime minister resigned because he threw a party during Covid. 

Or as it happened in Ukraine in 2014, the people did not like the fact that the president decided not to follow the previously planned path of integration with Europe. 

And only in countries where there is no democracy people think that they are nothing and everything is decided by their government. 

I understand that in russia, historically it is the tsar who decides everything, and with just a huge amount of resources people live very poorly. But this is exactly the choice of 140 million people. And it is their right to live that way. 

But let them live this way in their own country and stay out of our way. 

And if they do, they will all be to blame, even if they did nothing. 

In fact, this is what we are witnessing now.

**Comment by Dmi3yy** [comment]
And to the question of how it relates to EVO and Opsors I can also answer. 

For the last 10 years I was very closely involved in the development of EVO, and also my work was closely related to it. But after the war started I had to close my company, because some of the guys went to war and some clients went bankrupt. So I had to solve the problems.  

For me the war is a tragedy not only for the country but also personally, I have already lost some friends and acquaintances and relatives because of it. I lost them forever because they died.  

And against the backdrop of this tragedy the community was divided into two parts: 
- Those who also suffered and those who understand that war is not normal. 
- Those who think it's not his business, and he is not responsible for anything.

So yes for me now the world is mostly divided into black and white. 

Translated with www.DeepL.com/Translator (free version)

**Comment by Dmi3yy** [comment]
It's also very funny to see the version: 
community edition of community edition. 

Because, in general, this repository is the main branch of community edition. I only nominally manage it and only because no one else wants to do it.

**Comment by Dreamer0x01** [comment]
> And against the backdrop of this tragedy the community was divided into two parts:
> 
>     * Those who also suffered and those who understand that war is not normal.
> 
>     * Those who think it's not his business, and he is not responsible for anything.

People are not clearly divided into two categories.
There are many more of them. For example, those who understand that war is terrible, but whose opinion no one took into account and did not even ask when making decisions. And now there are repeated attempts to shift responsibility to this category of people as well. No, they did not make these decisions, much less take responsibility for them.

**Comment by Dmi3yy** [comment]
Time will tell. I see no point in arguing and proving anything. 

As an example, the UN has already adopted a resolution on reparations, which shows that all russians will not receive them, regardless of whether they are for the war or against it, and in fact this again speaks in favor of the fact that collective responsibility is not just words. 

It is clear that everything is more complicated, but in life there are situations when there are only two answers: 
- YES 
- NO

**Comment by Dreamer0x01** [comment]
Yes, time will tell.
All this does not change the name of the link community in any way. Which is called exactly as it was written in the message - CE, not RE.

**Comment by Dmi3yy** [comment]
> Yes, time will tell.
> All this does not change the name of the link community in any way. Which is called exactly as it was written in the message - CE, not RE.

Yep you can use any name, but it doesn't change the essence. 

Аctually in russia it is so accepted:
the fact that the war was called a special military operation also does not change the essence

**Comment by Seiger** [comment]
Unfortunately, the russians steal everything they can get their hands on, and Evolution CMS was unfortunately no exception.

If you really want to support the development of the system, then use the official and not the stolen version.

**Comment by Dreamer0x01** [comment]
Firstly, what does nationality have to do with it? Anyone can steal or not steal, it's just a matter of decency, which has nothing to do with a person's nationality or citizenship.

Secondly, CE was formed as a community of free developers independent of politics. Who have done quite a lot since they started launching ships and sorting things out in this community.

**Comment by Seiger** [comment]
I support the thesis that a person's nationality or affiliation does not matter. But it doesn't work with the russians.

**Comment by 0test** [comment]
Ok. "If you really want to support the development of the system", then delete and do not use:
1. DocLister
2. Formlister, 
3. Commerce
4. Te3
5. ClientSettings
6. eFilter
7. EditDocs
....
because. these are russian and belarusian packages. From the "Extras" remove too.

 Half or more of the documentation, as I am its author (russian)
 Dont see youtube video (for the same reason)

Or "I support the thesis that a **packages** nationality or affiliation does not matter"?

**Comment by Dreamer0x01** [comment]
It's even hard to comment on. Sometimes you say that nationality is not important, and then suddenly you highlight one of the nationalities again with the following phrase.

**Comment by Dreamer0x01** [comment]
In general, it is at least strange to use such a concept as "stolen" in the open source world.

And dividing people by nationality is doubly strange.

**Comment by Seiger** [comment]
russians, do not get and sign Belarusians. All Belarusians I know hate you just as much as Ukrainians.

**Comment by 0test** [comment]
Even if it's true, it's off topic.

> russians, do not get and sign Belarusians. All Belarusians I know hate you just as much as Ukrainians.

**Comment by Grinyaha** [comment]
> russians, do not get and sign Belarusians. All Belarusians I know hate you just as much as Ukrainians.

You are a piece of shit. Who and from whom else that stole it is necessary to find out. Delete everything Russian from Evo and don't use it. And f#ck you!

**Comment by Dreamer0x01** [comment]
Это, конечно, очень круто, написать одну национальность с маленькой буквы, две другие - с большой :) А еще два комментария назад кто-то говорил, что согласен с тем, что национальность не имеет значения.

Противно это донельзя, когда начинается вот эта вот гадость, деление людей на равных и не равных, достойных и не достойных лишь потому, что им довелось родиться на другом клочке земли. И самое противное, что те, кто родились на другом, считают это каким-то своим достижнем, позволяющим унижать других.

Написал без перевода, так как не настолько силен в английском, и нюансы мысли побоялся исказить.

**Comment by Seiger** [comment]
Ой как тебя закусило, сопельки свои подотрите, и не думайте что ваши инструменты такие уж уникальные.

Мы то ваши инструменты випилим нахуй из движка. Вот только б не пиздили вы наш инструментарий.

А так похуй на вашу недонацию.

**Comment by Dmi3yy** [comment]
После того как украли то что бесплатно говорить об алекватности очень сложно. 

Но каждый сам для себя делает выбор что ему нужно. 

К примеру мухам мед не нужен, и для них говно лучше всего, так и с людьми. И тут вопрос не только к национальности, то что сейчас орки решили что они венец эволюции это их право, такое же как и мое право считать их хуже говна.  

Очень жаль что многие становятся на путь разрушения а не созидания….

**Comment by Dmi3yy** [comment]
И да очень важный момент: вопрос украли не в том что сделали форк а в том что украли доступ к репозиторию…. 

P.s. Понимаю что признавать свои ошибки или свою безграмотность очень сложно но это единственный способ стать лучше.

**Comment by Grinyaha** [comment]
> 

Сначала выпили, долбаеб, потом кнопки нажимай. А на счет нации ты просто смешен и обоссан, потому что ну как ты можешь знать все наши национальности, говна ты кусок? Ты нонейм, звать тебя никак, бери автомат в руки и восстанавливай справедливость, а не в чатах тут пизди какой ты воин диванный.

**Comment by Dreamer0x01** [comment]
> А так похуй на вашу недонацию.
Вы считаете каким-то своим достижением принадлежность к другой нации? 

> Очень жаль что многие становятся на путь разрушения а не созидания….
И это правильно сказано. Очень жаль, что созидательная деятельность, такая как создание хорошей CMS и уникального сообщества вокруг неё, было разрушено ненавистью, хамством и национальной неприязнью. 

> И да очень важный момент: вопрос украли не в том что сделали форк а в том что украли доступ к репозиторию…
Ваш коллега чуть выше написал, что украдена была именно EvolutionCMS. С чего собственно и начался этот новый виток спора.
